trace_decoder/src/lib.rs

```diff
@@ -123,7 +123,10 @@ pub struct BlockTrace {
 
     /// The code_db is a map of code hashes to the actual code. This is needed
     /// to execute transactions.
-    pub code_db: Option<HashMap<H256, Vec<u8>>>,
+    // We actually ignore the keys in this map,
+    // this format is only retained for compatibility reasons
```

Comment:
> this format is only retained for compatibility reasons

I don't think there is any outside restriction, as the `code_db` is built inside `zero_bin`. I think we could convert it to a `HashSet` directly there to get rid of the keys